keyboards/opticalkb/rev1/rules.mk

```diff
@@ -0,0 +1,30 @@
+# https://docs.qmk.fm/#/squeezing_avr?id=rulesmk-settings
+#
+
```

Comment:
```suggestion
```

Reply:
Thanks for suggestions. Requested changes are committed.